app.js

```diff
@@ -1,123 +1,236 @@
 'use strict';
 
 var storeHours = ['6am', '7am', '8am', '9am', '10am', '11am', '12pm', '1pm', '2pm', '3pm', '4pm', '5pm', '6pm', '8pm', 'total'];
-//15 hours in day
 
-// create ul tag
-// create one li tag
-// fill li tag innertext w/ information
-// append li tag to ul as child
-// append ul to body
-// var element = document.createElement(tagName[, options]);
```

Comment:
Great use of pseudocode to think through what you want to accomplish.   The smaller you can break down a problem into pieces the easier it will be to write logic and build more complex things.